### PR TITLE
fix model script and model instance

### PR DESCRIPTION
**Pull request by genric**

### CODE REVIEW

File: modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.1.json [hunk 1]
```diff
@@ -52,15 +52,19 @@
...
             {
               "path": "schema:mediaType",
+              "minCount": 0,
               "name": "Media type",
               "description": "",
               "in": [
-                "application/swc,application/neurolucida,application/neuroml"
+                "application/swc",
+                "application/neurolucida",
+                "application/h5",
```
Comment (jdcourcol) [inline modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.1.json]: application/h5 we should be more specific here. hdf5 is just the container. and there are other groups that want to use h5 for their morphology but with a different layout.

Comment (MFSY) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.1.json]: What do you propose as a more specific value ?

Comment (jdcourcol) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.1.json]: I don't recall the syntax for mimetypes, but something that mentions vendor/bbp and  /morphology/h5

Comment (MFSY) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.1.json]: The mimetype syntax are not definitely not correct here. If your point was to have correct syntax let put that in a different [issue](#62) and solve in a different PR

Comment (jdcourcol) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.1.json]: no, it is not about the syntax. it is about application/h5 that is not specific enough. there is nothing that says it is bbp specific and it is about morphology. h5 is too broad

Comment (MFSY) [reply modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.1.json]: Okay. Understood. Let tackle it in the corresponding issue #62.

**Pull request merged**